app/templates/components/explore/side-bar.hbs

```diff
@@ -0,0 +1,105 @@
+<div class="ui list">
```

Comment:
@CosmicCoder96 yes. For now. We'll decide how we will be hiding it once all other issues of this PR are solved :)